habitat_sim/utils/viz_utils.py

```diff
@@ -26,6 +31,260 @@ def is_notebook():
         return True
 
 
+def make_cfg(settings):
```

Comment:
Can the logic from https://github.com/facebookresearch/habitat-sim/blob/9c96776cf6a783d036f7d051d648864dd6f20a13/examples/settings.py#L38 . Maybe those utility functions should be merged into Habitat-Sim proper.

Reply:
Does this function really deserve to be in viz_utils? Should it be renamed to something else?